CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # business_calendar changes by version
 
+2.0.0
+---------
+
+- Breaking changes: may not work with Ruby 1.8.7. Tested on 2.5.1.
```

Comment:
The tests are still green on 1.8.7 in travis

Reply:
I will go down to version 1.1.0 and remove this from the changelog.